vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -308,31 +308,18 @@ def _make_page_404_layout(self):
         return html.Div(
             [
                 # Theme switch is added such that the 404 page has the same theme as the user-selected one.
-                html.Div(
-                    children=dbc.Switch(
-                        id="theme-selector",
-                        value=self.theme == "vizro_light",
-                        persistence=True,
-                        persistence_type="session",
-                    ),
-                    id="settings",
+                dbc.Switch(
+                    id="theme-selector",
+                    value=self.theme == "vizro_light",
+                    persistence=True,
+                    persistence_type="session",
                 ),
                 html.Img(src=f"data:image/svg+xml;base64,{error_404_svg}"),
-                html.Div(
-                    [
-                        html.Div(
-                            children=[
-                                html.H3("This page could not be found.", className="heading-3-600"),
-                                html.P("Make sure the URL you entered is correct."),
-                            ],
-                            className="error-text-container",
-                        ),
-                        dbc.Button(children="Take me home", href=get_relative_path("/")),
-                    ],
-                    className="error-content-container",
-                ),
+                html.H3("This page could not be found."),
+                html.P("Make sure the URL you entered is correct."),
+                dbc.Button(children="Take me home", href=get_relative_path("/"), className="mt-4"),
             ],
-            className="page-error-container",
+            className="d-flex flex-column align-items-center justify-content-center min-vh-100",
```

Comment:
This is the main change: We would use Bootstrap shortcuts Instead of custom `classNames`.

This approach has the advantage of allowing us to remove more custom CSS from our static folder. However, it also makes it more difficult for users to overwrite styles, as they can no longer target a specific `className` like `page-error-container`. Additionally, these Bootstrap shortcuts apply the CSS properties with an important tag, which makes customization even more difficult.

So, we need to decide on a case-by-case basis whether to implement this change. For example, we want users to easily target and overwrite styles for certain containers (e.g., control panel, left-side, right-side, grid-layout) using `classNames` or `IDs`, so I wouldn't change it to use BS shortcuts.

In this case, I believe users rarely need to overwrite the existing `page-error-container`. If they want to customize the 404 page, they can still modify the `_make_page_404_layout()` method. 

Let me know what you think @antonymilne. I also don't want this to get too complex. So if it's not clear when to use BS shortcuts vs. custom classNames, then I am also fine leaving it as is 😄

Reply:
Definitely agree with using bootstrap here.